aerosol_dynamics/particle_test.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+from aerosol_dynamics import particle
+from . import ureg
+
+particle1 = particle.Particle(name='particle1',
+                              radius=1.0e-9 * ureg.m,
+                              density=1.0 * ureg.kg / ureg.m**3,
+                              charge=1.0 * ureg.C,)
+
+particle2 = particle.Particle(name='particle2',
+                              radius=1.0e-7 * ureg.m,
+                              density=1.8 * ureg.kg / ureg.m**3,
+                              charge=1.0 * ureg.C,)
+
+particle3 = particle.Particle(name='particle3',
+                              radius=1.0e-7 * ureg.lb,
+                              density=1 * ureg.kg / ureg.m**3,
+                              charge=3,)
+
```

Comment:
Generally, I think this should be rewritten like this:

```python
particle2 = particle.Particle(
    name='particle2', # one can add comments like this
    radius=1.0e-7 * ureg.m,
    density=1.8 * ureg.kg / ureg.m**3,
    charge=1.0 * ureg.C, # I think we shouldn't have space before/after * and /
)
```

Also probably use double quotations, `"`, here and everywhere to be consistent; Python treats them the same.

Reply:
Single quotations are used consistently throughout my files because that's my style; it feels cleaner to me. I'll change it if others are used to double quotations. 

I think I prefer keeping the spaces between * and /, only to space it out more. It can become harder to read when units get complicated. We may need to cut out spaces in those cases and group units together

Reply:
We can go either way with quotes --- we should try to stick to one. I like the idea of double quotes because it is clearer, and it encourages me to use less of them, while a single quote can blend in weirdly with many other things.

For spacing, I would follow [this](https://www.python.org/dev/peps/pep-0008/#other-recommendations), which I find especially helpful when debugging and disentangling the order of operations

Reply:
That's fine, we can go with double quotes. I'll make those changes. 

The reference you pointed to is what I'll use when there are mixed order of operations, like * / and + -. Both PEP8 and the Google style guide are unclear about what to do in a sequence of * and /. It does feel a little strange having spaces given the lack of space around the =, so I'm amenable to removing them.

Reply:
No need to follow right away, just thoughts for now. We will see what other people think/prefer. It won't make much of a difference eventually. What I have been doing is just following whatever the defaults on VSC are saying, unless something that deeply bothers me 😄